prime-router/settings/staging/0024-hhsprotect.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: "BLOBSTORE"
+organizationName: "ignore"
+topic: "covid-19"
+translation: !<CUSTOM>
+  schemaName: "hhsprotect/hhsprotect-covid-19"
```

Comment:
Fixed a bug that this was sending waters.schema to the hhsprotect blobstore (in staging only - this receiver is not in prod).  Still, this was confusing to the HHSProtect team testing our data in Staging.